pkg/oc/bootstrap/docker/run_self_hosted.go

```diff
@@ -139,6 +127,18 @@ func (c *ClusterUpConfig) StartSelfHosted(out io.Writer) error {
 		}
 	}
 
+	// if we're supposed to write the config, we'll do that and then exit
```

Comment:
Rather than re-ordering here, can we move the static pod creation to the `BuildConfig` method, since we're treating it as config?

Reply:
good idea, moved.